stencil-library/src/components/billing-form/billing-form.scss

```diff
@@ -3,4 +3,52 @@
 :host {
   @include host-base-styles;
   display: block;
-}
\ No newline at end of file
+}
```

Comment:
For this component the implementer can actually use the css parts directly, because they are exported from billing form. So I think the css variables are only necessary for anything that uses the payment method components (because we need a way to grab the styles via javascript to send through the iframe). We should discuss. I think the CSS parts are a better experience and we should only use the css variables when we have to, but maybe that's confusing